test/integration/functional_test.go

```diff
@@ -155,23 +155,43 @@ func validateNodeLabels(ctx context.Context, t *testing.T, profile string) {
 func validateDockerEnv(ctx context.Context, t *testing.T, profile string) {
 	defer PostMortemLogs(t, profile)
 
-	mctx, cancel := context.WithTimeout(ctx, Seconds(13))
+	mctx, cancel := context.WithTimeout(ctx, Seconds(30))
 	defer cancel()
-	// we should be able to get minikube status with a bash which evaled docker-env
-	c := exec.CommandContext(mctx, "/bin/bash", "-c", "eval $("+Target()+" -p "+profile+" docker-env) && "+Target()+" status -p "+profile)
-	rr, err := Run(t, c)
+	var rr *RunResult
+	var err error
+	if runtime.GOOS == "windows" { // golang exec powershell needs some tricks !
```

Comment:
This comment is not helpful in this context. Remove it.

Reply:
done